charts/stable/autobrr/values.yaml

```diff
@@ -0,0 +1,87 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: ghcr.io/autobrr/autobrr
+  # @default -- chart.appVersion
+  tag:
+  # -- image pull policy
+  pullPolicy: IfNotPresent
+
+# -- environment variables. See [image docs](https://docs.k8s-at-home.com/our-container-images/configuration/) for more details.
+# @default -- See below
+env:
+  # -- Set the container timezone
+  TZ: UTC
+
+# See https://raw.githubusercontent.com/autobrr/autobrr/develop/config.toml
+config: |
+  # config.toml
+
+  # Hostname / IP
+  #
+  # Default: "localhost"
+  #
+  host = "0.0.0.0"
+
+  # Port
+  #
+  # Default: 7474
+  #
+  port = 7474
+
+  # Base url
+  # Set custom baseUrl eg /autobrr/ to serve in subdirectory.
+  # Not needed for subdomain, or by accessing with the :port directly.
+  #
+  # Optional
+  #
+  #baseUrl = "/autobrr/"
+
+  # autobrr logs file
+  # If not defined, logs to stdout
+  #
+  # Optional
+  #
+  #logPath = "log/autobrr.log"
+
+  # Log level
+  #
+  # Default: "DEBUG"
+  #
+  # Options: "ERROR", "DEBUG", "INFO", "WARN"
+  #
+  logLevel = "INFO"
+
+  # Session secret
+  #
+  sessionSecret = "secret-session-key"
+
```

Comment:
Secret key should be rotated.

Reply:
Example of session key: https://github.com/swizzin/swizzin/blob/3e29425507ef6711cd36791c008490ecf32689df/scripts/install/autobrr.sh#L52